src/Background/Animation.vala

```diff
@@ -17,20 +17,20 @@
 
 namespace Gala {
     public class Animation : Object {
-        public string filename { get; construct; }
-        public string[] key_frame_files { get; private set; default = {}; }
+        public GLib.File file { get; construct; }
+        public GLib.GenericArray<GLib.File> key_frame_files { get; private set; default = new GLib.GenericArray<GLib.File> (); }
```

Comment:
Does this need to be a GenericArray? given that we known that the max number of files we will use is 2, isn't a fixed array enough.